docs/development/releasing.rst

```diff
@@ -12,231 +12,249 @@ more of the process, if not all.
+   changes in the helpers.  See the section :ref:`helpers-release-info` for
+   more on this.
 
-        $ git shortlog -s
+#. Make sure that the continuous integration services (E.g., Travis) are passing
+   for the `astropy core repository`_ branch you're going to release.  You may
+   also want to locally run the tests in ``remote-data`` mode, as those are not
+   necessarily run automatically::
 
-    And just add anyone from that list who isn't already credited.
+      $ python setup.py test --remote-data
 
- 3. Install virtualenv if you don't already have it.  See the linked virtualenv
-    documentation for details.  Also, make sure that you have `cython`_
-    installed, as you will need it to generate the .c files needed for the
-    release.
+#. Ensure you have a GPG key pair available for when git needs to sign the
+   tag you create for the release.  See :ref:`key-signing-info` for more on
+   this.
 
- 4. Create and activate a virtualenv::
+#. Obtain a *clean* version of the `astropy core repository`_.  That is, one
+   where you don't have any intermediate build files.  Either use a fresh
+   ``git clone`` or do ``git clean -dfx``.
 
-        $ virtualenv --system-site-packages astropy-release
-        $ source astropy-release/bin/activate
+#. Be sure you're on the branch appropriate for the version you're about to
+   release.  For example, if releasing version 0.2.2 make sure to::
 
- 5. Obtain a *clean* version of the Astropy repository.  That is, one
-    where you don't have any intermediate build files.  Either use a fresh
-    ``git clone`` or do ``git clean -dfx``.
+      $ git checkout v0.2.x
 
- 6. Be sure you're the "master" branch or, for a bug fix release, on the
-    appropriate bug fix branch.  For example, if releasing version 0.2.2 make
-    sure to::
+#. Edit the ``setup.py`` file by removing the ``".dev"`` at the end of the
+   ``VERSION`` string.
 
-        $ git checkout v0.2.x
+#. Edit the ``CHANGES.rst`` file by changing the date for the version you are
+   about to release from "unreleased" to today's date.  Also be sure to remove
+   any sections of the changelog for that version that have no entries.
 
- 7. Now install Astropy into the virtualenv::
+#. Add the changes to ``CHANGES.rst`` and ``setup.py``::
 
-        $ python setup.py install
+      $ git add CHANGES.rst setup.py
 
-    This is necessary for two reasons.  First, the entry points for the
-    releaser scripts need to be available, and these are in the Astropy
-    package. Second, the build process will generate .c files from the
-    Cython .pyx files, and the .c files are necessary for the source
-    distribution.
+   and commit with message::
 
- 8. Install zest.releaser into the virtualenv; use ``--upgrade --force`` to
-    ensure that the latest version is installed in the virtualenv (if you're
-    running a csh variant make sure to run ``rehash`` afterwards too)::
+      $ git commit -m "Preparing release v<version>"
```

Comment:
So actually it looks like historically we don't include ``v`` in the version in the 'Preparing' and 'Back to development' commits